### PR TITLE
Upgrade vtk.js version

### PR DESCRIPTION
**Pull request by annehaley**

### CODE REVIEW

File: web_client/src/store/index.ts [hunk 1]
```diff
@@ -227,9 +229,11 @@ function progressHandler(completed, total) {
 }
 
 /** Creates array of tasks to run then runs tasks in parallel. */
-function startReaderWorkerPool() {
+function startReaderWorkerPool(onDownloadProgress) {
   // Get the current array of tasks in readDataQueue
-  const taskArgsArray = readDataQueue.map((taskInfo) => [taskInfo]);
+  const taskArgsArray = readDataQueue.map((taskInfo) => [
+    Object.assign(taskInfo, { onDownloadProgress }),
```
Comment (zachmullen) [inline web_client/src/store/index.ts]: Should we do the spread operator here, or do we have to do `Object.assign` to modify in-place?

File: web_client/src/types.ts [hunk 2]
```diff
@@ -183,6 +183,7 @@ interface MIQAStore {
   currentFrameId: string | null;
   loadingFrame: boolean;
   errorLoadingFrame: boolean;
+  onDownloadProgress: (e: Event) => null | null;
```
Comment (zachmullen) [inline web_client/src/types.ts]: Should this be `(e: Event) => void` instead? (I am not a typescript expert so I'm not sure)

**Review by zachmullen — APPROVED** [review]
LGTM

File: web_client/src/store/index.ts (hunk 1 above)
Comment (annehaley) [reply web_client/src/store/index.ts]: The spread operator suffices here, so I changed it in [ded9674](https://github.com/OpenImaging/miqa/pull/657/commits/ded9674a438a8b0663cb19b748faaa0dcf107ce8)

File: web_client/src/types.ts (hunk 2 above)
Comment (annehaley) [reply web_client/src/types.ts]: Yes, you're right. I changed it in [9bb7713](https://github.com/OpenImaging/miqa/pull/657/commits/9bb7713fe1ad7c76d76d54829f5afb1b73072fdf)

**Pull request merged**